tools/query_breakdown/src/main/java/com/google/bigquery/Node.java

```diff
@@ -7,22 +9,23 @@
 public class Node {
 
   // ensures tree structure
-  ArrayList<Node> children;
-  Node parent;
+
+  private ArrayList<Node> children;
+  private Node parent;
 
   // ensures that we know which location the error occurs in
-  int startLine;
-  int startColumn;
-  int endLine;
-  int endColumn;
+  private int startLine;
+  private int startColumn;
+  private int endLine;
+  private int endColumn;
 
   // ensures that we know how the error was handled
-  boolean errorHandlingType;
-  String replaceFrom;
-  String replaceTo;
+  private boolean errorHandlingType;
```

Comment:
Use an enum instead of boolean for `errorHandlingType`.

Reply:
Hopefully resolved with the most recent commit!